### PR TITLE
Dark mode!!

### PR DESCRIPTION
**Pull request by Wolfsurge**

### CODE REVIEW

File: frontend/src/pages/Upload.css [hunk 1]
```diff
@@ -1,17 +1,67 @@
...
+}
+
+.page #heading {
+	color: white;
+}
+
+input[type=file]::file-selector-button {
+	margin-right: 20px;
+	border: none;
+	background: hsla(34, 60%, 57%, 1);
```
Comment (nickykitchingman) [inline frontend/src/pages/Upload.css:13]: `var(--color-1)`?

File: frontend/src/pages/Upload.css [hunk 2]
```diff
@@ -1,17 +1,67 @@
...
+	padding: 10px 20px;
+	border-radius: 10px;
+	color: #fff;
+	cursor: pointer;
+	transition: background .2s ease-in-out;
+}
+
+input[type=file]::file-selector-button:hover {
+    background-color: hsla(214, 60%, 57%, 1);
```
Comment (nickykitchingman) [inline frontend/src/pages/Upload.css:22]: `var(--color-2)`?

File: frontend/src/styles.css [hunk 3]
```diff
@@ -111,15 +111,15 @@
     --color-1-lightest             : hsla(34, 60%, 97%, 1);
     --color-1-lighter              : hsla(34, 60%, 77%, 1);
     --color-1-light                : hsla(34, 60%, 67%, 1);
-    --color-1-dark                 : hsla(34, 60%, 47%, 1);
+    --color-1-dark                 : rgb(30, 30, 35);
```
Comment (nickykitchingman) [inline frontend/src/styles.css:114]: Maybe keep this as hsla for consistency? The reason being is that their all the same as either `color-1` or `color-1`, with different *light*s - different shades.

**Review by nickykitchingman — CHANGES_REQUESTED** [review]
Lil tweaks here n there

File: frontend/src/components/StatusSymbol.css [hunk 4]
```diff
@@ -1,24 +1,29 @@
 .status-pending {
+	background-color: var(--color-btn);
```
[the remark this answers is not in the document]
Comment (Wolfsurge) [reply frontend/src/components/StatusSymbol.css:2]: They will default to the same colour as the page background

**Review by nickykitchingman — APPROVED** [review]
Looking real nice now

**Pull request merged**